src/domain/harbors/HarborsPageContainer.tsx

```diff
@@ -59,69 +62,107 @@ const HarborsContainer: React.FC = () => {
       Header: t('harbors.tableHeaders.places'),
       accessor: 'numberOfPlaces',
     },
+    {
+      Cell: ({ cell }) => (
+        <div>
+          <IconExclamationCircle
+            className={classNames(styles.icon, styles.nowrapper, {
+              [styles.disabled]: !cell.value,
+            })}
+          />
+        </div>
+      ),
+      Header: () => (
+        <div className={classNames(styles.icon, styles.nowrapper)}>
```

Comment:
In general, we should avoid tweaking the styles in containers, but rather in presentational components.

Reply:
I agree. All the presentational stuff seems to be in the container here. The next PR moves all presentational stuff out of the container for storybook support.